lib/__tests__/overrides.test.mjs

```diff
@@ -127,6 +127,21 @@ describe('single input file. all overrides are matching', () => {
 		expect(linted.results[0].warnings[1].rule).toBe('color-named');
 	});
 
+	it('override with extends in config property', async () => {
+		const { results } = await standalone({
+			files: [path.join(fixturesPath, 'style.css')],
+			config: {
+				extends: path.join(fixturesPath, 'extends-in-overrides.json'),
```

Comment:
[note] Here's the content of `extends-in-overrides.json`:
https://github.com/stylelint/stylelint/blob/baf7c0e4448e6e1b1d472b53df8a3dd35291449a/lib/__tests__/fixtures/config-overrides/extends-in-overrides.json#L1-L8